src/MICore/UnixUtilities.cs

```diff
@@ -42,6 +42,12 @@ public static class UnixUtilities
             string pidFifo,
             string debuggerCmd)
         {
+            // On OSX, 'wait' will return once there is a status change from the launched process rather than for it to exit, so
+            // we need to use 'fg' there. 
+            // On Linux, fg will fail with 'no job control' because our bash shell is not interactive, so we need 
```

Comment:
Do you happen to know why this doesn't happen on OS X?

I tried using `fg` when I first added this. I tried it on OS X and it worked (as you have seen since you're adding that), and I thought I had tried on Linux too, but maybe I'm misremembering.